core-features/generate-client-and-cli-install-no-lockfile-npm/run.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+set -eux
+
+VERSION=$(cat ../../.github/prisma-version.txt)
+
+cp -r ../_common/generate-client-and-cli-install/* .
+
+# we remove the lockfile to make sure that the install works without it
+# when no lockfile is found in a project, we will use npm for installs
+# currently, the root lockfile interferes with the detection, remove it
+rm ../../pnpm-lock.yaml
+rm package-lock.json
```

Comment:
huh? is that a bug?

Reply:
Our workspace is a pnpm one, so it gets correctly detected as such.

Reply:
But what I want to test is that `npm` is used when there is no lockfile.